contracts/Portfolio.sol

```diff
@@ -121,28 +122,8 @@ abstract contract PortfolioVirtual is Objective {
         return __account__.reserves[token];
     }
 
-    /// @inheritdoc IPortfolioGetters
-    function getBalance(
-        address owner,
-        address token
-    ) public view returns (uint256) {
-        return __account__.balances[owner][token];
-    }
-
     // ===== External Actions ===== //
 
-    /// @inheritdoc IPortfolioActions
-    function deposit() external payable override lock {
```

Comment:
this drops support for directly using ETH in the protocol, right? so users need to wrap to WETH first.
But for each WETH pool you're paying out in ETH. Might make sense to completely drop ETH support now?

Reply:
It's still supported, in the beginning of `multiprocess` the internal `_deposit` is called. This only drops support for carrying balances over between two+ transactions

Reply:
right, forgot about the call at the beginning of multiprocess